jena-arq/src/main/java/org/apache/jena/riot/lang/rdfxml/SysRRX.java

```diff
@@ -29,8 +29,21 @@
  */
 public class SysRRX {
 
+    /**
+     * Creates and initializes a javax.xml.stream.XMLInputFactory#newDefaultFactory().
+     * @return
+     */
     public static XMLInputFactory createXMLInputFactory() {
-        XMLInputFactory xmlInputFactory = XMLInputFactory.newInstance();
+        return initAndConfigure(XMLInputFactory.newInstance());
+    }
+
+    /**
+     * Configures the parser to be safe and sets necessary properties.
+     * This method should be called when another factory than javax.xml.stream.XMLInputFactory#newDefaultFactory()
+     * shall be used.
+     * @param xmlInputFactory
```

Comment:
I recognized, that I kept using `javax.xml.stream.XMLInputFactory#newInstance()` instead of `javax.xml.stream.XMLInputFactory#newDefaultFactory()`, so I fixed the javadoc accordingly.